frontend/rust-lib/flowy-plugins/src/entities.rs

```diff
@@ -0,0 +1,81 @@
+use flowy_derive::ProtoBuf;
+use flowy_error::ErrorCode;
+
+/*
+ model="text-davinci-003",
+ prompt="Write a tagline for an ice cream shop."
+*/
+#[derive(Default, ProtoBuf)]
+pub struct TextCompletionPayloadPB {
+  #[pb(index = 1)]
+  pub request_id: String,
+
+  // Model: Either text-davinci-003 or gpt-3.5-turbo
+  #[pb(index = 2)]
+  pub model: String,
+
+  // Prompt to query gpt
+  #[pb(index = 3)]
+  pub prompt: String,
```

Comment:
Actually, I think there's no need to pass the `open_ai_key` and the model. We can have a settings page for OpenAI to handle this. I will update the logic here after merging this PR.

Reply:
Sure, i don't need to make any changes for this part of code right ??